src/pages/julieLee/Login/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import "./Login.scss";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+
+function LoginJulie() {
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate("/main-julie");
+  };
+
+  const [identify, setIdentify] = useState("");
+  const handleIdInput = (event) => setIdentify(event.target.value);
+  const [password, setPassword] = useState("");
+  const handlePwInput = (event) => setPassword(event.target.value);
+
+  const validation = (idText, pwText) => {
+    if (idText.includes("@") && pwText.length >= 5) {
+      return true;
+    }
+  };
```

Comment:
```suggestion
    }
    
    return false;
  };
```

return 타입 명시적으로 해주세요.